transport.go

```diff
@@ -6,12 +6,19 @@ import (
 
 	peer "github.com/libp2p/go-libp2p-peer"
 	tpt "github.com/libp2p/go-libp2p-transport"
+	transport "github.com/libp2p/go-libp2p-transport"
 	smux "github.com/libp2p/go-stream-muxer"
 	ma "github.com/multiformats/go-multiaddr"
 	mafmt "github.com/multiformats/go-multiaddr-fmt"
 	"github.com/pions/webrtc"
 )
 
+// Ensure that our Transport implements the correct interface.
+var _ transport.Transport = &Transport{}
+
+// Fmt is the Multiaddress format for WebRTC
+var Fmt = mafmt.And(mafmt.HTTP, mafmt.Base(Protocol.Code))
```

Comment:
For now, we're only supporting signaling over HTTP. In the future it should be possible to support many different transports. Hopefully we can rely on existing code in libp2p to set up a connection (likely a `Stream`) using an arbitrary transport.